### ISSUE TITLE
Overall impression and interpreted Java

### ISSUE THREAD

**Issue by igouy** [attractivechaos/plb2#50]
> interpret[t]ed with no compilation
> …
> compile hot code on the fly and then execute

You could contrast jit and interpreted directly by showing timings for both default `java` and `java -Xint`.

**Comment by attractivechaos** [comment]
Good idea. I believe node can be run in `--jitless` mode (inherited through v8):
```
--jitless                   disable runtime allocation of executable memory
```
At least with this option, node becomes much slower. I will probably add this and also [quickjs](https://bellard.org/quickjs/).

**Comment by igouy** [comment]
https://v8.dev/blog/jitless

I don't pretend to understand `--jitless` and `--no-opt`

**Comment by attractivechaos** [comment]
The timing with `--jitless` vs with `--jitless --no-opt` is nearly identical. I will assume `--jitless` turns off the JIT completely. Fabrice Bellard seems to do the same when evaluating v8.
